app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```

Comment:
This commit adds an empty module to the project. It is not very useful.

Your next commit adds a login helper, which is very useful. I think this commit should be squashed with the one after it.

Remember: Commits need to be **functional**